camera-orderfile/README.md

```diff
@@ -0,0 +1,95 @@
+# NdkCamera Sample
```

Comment:
Do we actually need the second sample? This sample is _big_ and I'm not sure what it shows that the dedicated `ordersample` doesn't. If it's not going to teach anything new it's better to delete it (otherwise readers will, same as I am, spend time trying to figure out what the point is). If it is just Java vs Kotlin, better to copy the orderfile example and translate that to the other language :)